components/rewards/RewardTab.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  RewardTabContainer,
+  RewardDescriptionContainer,
+  StarIcon
+} from '../../styled/rewards';
+import { Subhead, Caption } from '../BaseComponents';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class RewardTab extends React.Component {
+  render() {
+    return (
+      <RewardTabContainer>
```

Comment:
Rename to RewardCardContainer

Reply:
I know you used this name elsewhere, but you're probably getting rid of that once you change this all to flex right..?